lib/Wikia/tests/Service/User/Auth/MediaWikiAuthServiceTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Wikia\Service\User\Auth;
+
+use Wikia\Service\Helios\ClientException;
+use Wikia\Service\Helios\HeliosClient;
+
+class MediaWikiAuthServiceTest extends \WikiaBaseTest {
+	private $authService;
+	private $heliosClientMock;
+
+	public function setUp() {
+		parent::setUp();
+
+		$this->heliosClientMock = $this->getMock( HeliosClient::class,
+			[ 'login', 'forceLogout', 'invalidateToken', 'register', 'info', 'generateToken' ] );
+
+		$this->authService = new MediaWikiAuthService( $this->heliosClientMock );
+	}
+
+	public function testAuthenticateAuthenticationFailed() {
+		$username = 'SomeName';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SomeName" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)